smart-contracts/wasm-chain-integration/src/v1/trie/tests.rs

```diff
@@ -144,6 +144,41 @@ fn prop_insert_freeze_lookup() {
     QuickCheck::new().tests(NUM_TESTS).quickcheck(prop as fn(_, _) -> anyhow::Result<()>);
 }
 
+#[test]
+/// Check that migration works.
+fn prop_migration_retains_semantics() {
+    let prop = |inputs: Vec<(Vec<u8>, Value)>| -> anyhow::Result<()> {
+        // let inputs = inputs.into_iter().map(|(k, mut v)| {v.truncate(11); (k,
```

Comment:
Think it's safe to remove the commented code.

Reply:
Good catch. The history here is that I had a bug in migration that worked for "small" values, and I used this line to confirm where the issue was :P  Removed.